MetaMask/metamask-extension
# Fix portfolio URL configuration

The portfolio URL added in #15407 was meant to be configurable via environment variable or the `.metamaskrc` file, but that configuration was broken. Instead the default value was always used.

That configuration has been fixed. The portfolio URL can be set either by environment variable or configuration file.

## Manual Testing Steps

Set the portfolio URL via environment variable or the `.metamaskrc` file, and see that it is set correctly on the Home screen.

## Pre-Merge Checklist

  - An e2e test for the portfolio feature could have caught this. That seems like a lot of work to validate an isolated configuration bug, and I don't have time personally to work on that now, but [I have added an issue](https://github.com/MetaMask/metamask-extension/issues/15850) to track that we need an e2e test for this feature.

### + If there are functional changes:
